### ISSUE TITLE
[Feature Request] Apply patch show conflicts & staging option instead of instantly commiting

### ISSUE THREAD

**Issue by OzzysDen** [ForkIssues/TrackerWin#2389]
When applying a patch from top menu (Repository/Apply Patch), it commits. It would be nice to stage the files first, then choose/edit and see if there is any conflict. I know it can be done by commands but gui makes things easier.

**Comment by pytlus93** [comment]
Yes, would be great if it have similar functionality as SourceTree.

<img width="593" height="79" alt="Image" src="https://github.com/user-attachments/assets/a2340bae-0975-471b-be28-decdc6c7c692" />

Or at least variant with execution of something like this:
`git -c diff.mnemonicprefix=false -c core.quotepath=false --no-optional-locks apply -v --reject -p 1 patch_file.patch`
